### ISSUE TITLE
English language strings scream for a cleanup

### ISSUE THREAD

**Issue by MarcinOrlowski** [logisim-evolution/logisim-evolution#847]
While working on #840 I noticed that original (English) strings are of moderate quality. Major issues I spotted

* Lack of string context: That's a big problem for translators are unless they know the app inside out and all strings by heart, they cannot really tell from the key if this string is ie. a button and should be short or maybe it can be longer. Where it is used. We should ass some more comments than `options.java` to each section so translators no longer would need to guess (and sometimes miss)
* Quotation marks: sometimes it's `'` sometimes it is `"`  (I even more "sophisticated" mixes, fixed in #845)
* Capitalization: Sometimes `It Goes Like This`, while in other cases it `Goes like this`.
* Punctuation: some sentences end with `.` while other won't. Clear rules would be useful to have.
* Some strings are not really clear what they are all about
* Duplicates (i.e. `gui.properties` and `file.properties`) have exactly the same keys and values for multiple strings (I am going to see if that just copy-paste issue)
* Some texts are simply OLD (i.e. there's one that ask to let Carl Burch know if some type of error occur :) 
* Long string formatting: sometimes it is bungs  or separate keys (later concatenated in code) while other are just long strings using `\n` or `|`. This is a translators' nightmare.
* Some help (tooltip) texts say that given button "Edit this and that" but other would "Edits this". 
* All placeholders use formatting sequences (i.e. `%s`/`%d`) without using position index. (i.e. `%1$s`). That would help translators swapping the order if needed.
* Hardcoded names (I believed we removed most of hardcoded strings like app name etc, but there're still some)
* I am not a native but I'd bet some money that some strings are definitely in English-ish-alike-dialect :)

I just stopped PL works for now and I believe these steps are necessary to clean-up base strings:

- [ ] Look for duplicates  (I'll do it)
- [ ] Look for hardcoded name (I'll do it)
- [ ] Add placeholder position indices
- [ ] Capitalization clean-up
- [ ] Punctuation cleanup (I'll do it as prop-tool can find that)
- [ ] Define clear rules how strings should be constructed?
- [ ] English proofreading (best, by native, @davidhutchens ?)

@maehne @BFH-ktt1

**Comment by maehne** [comment]
@MarcinOrlowski: I agree with your observations. There's certainly some clean-up potential in the localisation files. They have grown over the years and were touched by various people, who naturally did not always have a complete overview of the code base. Any PRs improving this situation are welcome, but we need to make sure that we don't break the other translations.
